src/Microsoft.Extensions.WebEncoders/Testing/HtmlTestEncoder.cs

```diff
@@ -30,6 +51,21 @@ public override void Encode(TextWriter output, char[] value, int startIndex, int
 
         public override void Encode(TextWriter output, string value, int startIndex, int characterCount)
         {
+            if (output == null)
+            {
+                throw new ArgumentNullException(nameof(output));
+            }
+
+            if (value == null)
+            {
+                throw new ArgumentNullException(nameof(value));
+            }
+
+            if (value.Length == 0)
```

Comment:
Done. Note this particular comment wasn't hidden because there's no `characterCount` parameter here.